packages/online/src/useOnline.ts

```diff
@@ -29,12 +29,16 @@ function useOnline(): boolean {
   // we only needs this to be set on mount
   // hence []
   useEffect(() => {
-    window.addEventListener("online", setOnline);
-    window.addEventListener("offline", setOffline);
-    return () => {
-      window.removeEventListener("online", setOnline);
-      window.removeEventListener("offline", setOffline);
-    };
+    if (typeof window === 'undefined') {
```

Comment:
We need to put this inside the `getIsOnline` function and return `null` on the server side.